packages/ember-data/tests/unit/states-test.js

```diff
@@ -87,3 +87,14 @@ test("the deleted.saved state", function() {
   isFalse("isSaving");
   isTrue("isDeleted");
 });
+
+module("unit/states - hooks for record states", {
+  setup: function() {
+    rootState = DS.RootState;
+  }
+});
+
+test("root.deleted.uncommitted supports pushedData", function() {
+  equal(typeof get(rootState, "deleted.uncommitted.pushedData"), 'function',
+    "root.deleted.uncommitted should be a function");
+});
```

Comment:
Yep good call, will do. I didn't like this approach but wasn't sure how to correctly test it.